src/selectors/per-thread/markers.js

```diff
@@ -305,26 +390,16 @@ export function getMarkerSelectorsPerThread(threadSelectors: *) {
       return null;
     }
 
-    const marker = getMarker(selectedMarkerIndex);
-    if (!marker) {
-      console.error(stripIndent`
-        Couldn't find the selected marker index ${selectedMarkerIndex} in the full marker list.
-        This shouldn't normally happen and is likely a programming error.
-      `);
-
-      return null;
-    }
-
-    return marker;
+    return getMarker(selectedMarkerIndex);
```

Comment:
`getMarker` already throws if the marker isn't found, so I removed it here. I believe this was a left-over from a previous code.